source/fundamentals/serialization/guid-serialization.txt

```diff
@@ -24,87 +24,104 @@ In this guide, you can learn how to serialize **globally unique identifiers**
+         77665544-3322-1100-ffee-ddccbbaa9988
 
-- The ``BsonBinaryReader.ReadBinaryData()`` method ignores ``readerSettings.GuidRepresentation``
-- The ``BsonBinaryWriter.WriteBinaryData()`` method ignores ``writerSettings.GuidRepresentation``
-- The ``JsonReader.ReadBinaryData()`` method ignores ``readerSettings.GuidRepresentation``
-- ``JsonWriter`` ignores ``writerSettings.GuidRepresentation``
-- Calling the ``BsonBinaryData.ToGuid()`` method without the ``GuidRepresentation``
-  parameter works only on GUIDs of subtype 4.
+To standardize GUID byte order, we added ``BsonBinaryData`` subtype 4, which all
+MongoDB drivers encode in the same way. We recommend using 
+``BsonBinaryData`` subtype 4 for all new GUIDs.
 
-.. note::
+For a list of all ``BsonBinaryData`` subtypes, see the
+API documentation for the `BsonBinarySubType <{+new-api-root+}/MongoDB.Bson/MongoDB.Bson.BsonBinarySubType.html>`__
+enum.
+
+.. tip::
 
-   You can't use both ``GuidRepresentationMode.V2`` and ``GuidRepresentationMode.V3`` 
-   in a single application.
+   In MongoDB applications, ``ObjectId`` can be used as a unique identifier for
+   a document. Consider using ``ObjectId`` in place of a GUID with MongoDB
+   applications where possible.
 
-Serializing GUIDs in V3
------------------------
+Serializing GUIDs
+-----------------
 
-``GuidRepresentationMode.V3`` handles GUID serialization at the level of individual
-properties. This mode is more flexible than ``V2``, but it also means you must ensure that 
-each GUID field is serialized and deserialized correctly.
+Although we recommend using subtype 4 for all new ``BsonBinaryData`` GUIDs, some older
+MongoDB collections might contain some GUID fields that use subtype 3 and others that use
+subtype 4. To account for these differences, the {+driver-short+} handles GUID
+serialization at the level of individual properties.
 
-If you're using the {+driver-short+} to :ref:`automap your {+language+} classes to document schemas <csharp-class-mapping>`,
-you can use the ``BsonGuidRepresentation`` attribute on a GUID property 
-to specify the representation:
+If you're using the {+driver-short+} to
+:ref:`automap your {+language+} classes to document schemas <csharp-class-mapping>`,
+you can add the ``BsonGuidRepresentation`` attribute to a GUID property 
+to specify its representation. This attribute accepts a value from the 
+``GuidRepresentation`` enum, which contains the following members:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 10 10 
+
+   * - GuidRepresentation Member
+     - BsonBinaryData Subtype
+   
+   * - ``Standard``
+     - 4
+   
+   * - ``CSharpLegacy``
+     - 3
+   
+   * - ``JavaLegacy``
+     - 3
+   
+   * - ``PythonLegacy``
+     - 3
+   
+   * - ``Unspecified``
+     - N/A
+
+.. note::
+   
+   The ``CSharpLegacy``, ``JavaLegacy``, and ``PythonLegacy`` GUID representations are
+   all equivalent to ``BsonBinaryData`` subtype 3, but use different byte orders.
+
```

Comment:
S: break out the following content into a sub section

Reply:
I did a little reorganization, which I think matches your suggestion and improves the overall flow